tools/depends/native/cargo-c/CARGO-C-VERSION

```diff
@@ -0,0 +1,5 @@
+APPNAME=cargo-c
+BASE_URL=https://github.com/lu-zero/cargo-c/archive
+VERSION=v0.9.20
+ARCHIVE=$(VERSION).tar.gz
```

Comment:
Missing file name in `android-tools/xbmc-tarballs`

To follow the "pattern":

```
APPNAME=cargo-c
VERSION=v0.9.20
SOURCE=$(APPNAME)-$(VERSION)
ARCHIVE=$(SOURCE).tar.gz
```

Same in others

Reply:
Not sure where SOURCE is used but since we download off of tags, this won't work as the archive is only named by version.

Reply:
Maybe I have not explained myself correctly and also Linux depends build is not my thing but what I want to say is that all the other file names have the package name and the version and I don't think it has to be different for rust and cargo:

![tarballs](https://github.com/xbmc/xbmc/assets/58434170/44bf55df-6517-4ce0-adf7-d777fe04e145)

> Not sure where SOURCE is used

`SOURCE` is only a aux. variable to construct filename with `APPNAME`-`VERSION` format

Maybe @fuzzard can explain better

Reply:
As example all others follow same pattern:

https://github.com/xbmc/xbmc/blob/4dcaf8634c8519fb946557704831a991205955f1/tools/depends/native/cmake/Makefile#L5-L9

Reply:
I understand but you suggested changing `ARCHIVE`, which will break downloading off of GitHub tags.  

Other dependencies are uploaded to the Kodi source repo so they are not the same.
They don't use `BASE_URL`.

Reply:
I understand now. So probably until it is merged and the sources are uploaded to mirrors (and properly named), it cannot be done. Forget it for now.